tqp/newrelic.py

```diff
@@ -0,0 +1,37 @@
+import json
+import logging
+
+import newrelic.agent
+
+# -----------------------------------------------------------------------------
+
+
+def install():
+    QueuePollerBase._handle_message = newrelic.agent.background_task()(
+        QueuePollerBase._handle_message,
+    )
+
+    super_handle_message = TopicQueuePoller.handle_message
+    super_handle_error = TopicQueuePoller.handle_error
+
+    def _hook_handle_message(self, raw_msg, payload):
```

Comment:
probably name these just as the names of the overridden methods

you may also want to use `@functools.wraps`